source/opt/instruction.h

```diff
@@ -473,6 +495,11 @@ class Instruction : public utils::IntrusiveNodeBase<Instruction> {
   // empty.
   std::vector<Instruction> dbg_line_insts_;
 
+  // Debug scope information. If this Instruction is included in a scope,
+  // |dbg_scope_| must have non-zero |lexical_scope|. Otherwise,
+  // |dbg_scope_.lexical_scope| must be 0.
```

Comment:
The comments of the special values for debug scope are best placed on the definition of `DebugScope`.